client/lib/css-hot-reload/index.js

```diff
@@ -0,0 +1,87 @@
+/**
+ * External dependencies
+ */
+var io = require( 'socket.io-client' );
+
+/**
+ * Internal dependencies
+ */
+import { cssBuildFailed, cssBuilding } from 'state/application/actions';
+import i18n from 'lib/mixins/i18n';
+
+/**
+ * @returns {boolean} 
+ */
+function isChanged( href, changedFiles ) {
+	// "/calypso/style-debug.css?v=5a1db7fee7" -> "/calypso/style-debug.css"
+	var css = href.split('?')[0];
+	var i, re;
+		
+	for( i = 0; i < changedFiles.length; ++i ) {
+		re = new RegExp( '/' + changedFiles[i] + '$' );
```

Comment:
Is there a need for a `RegExp` here? It seems like it's overkill for the job. If we _know_ we have this structure, why not simply split and do a basic string search?

```js
import includes from 'lodash/includes'

// "/calypso/style-debug.css?v=5a1db7fee7" -> "style-debug.css"
const path = href.split( '?' ).shift().split( '/' ).pop();

return changedFiles.some( file => file === path )
```